v2alpha1/api/swagger/components/schemas/HostConnection.yaml

```diff
@@ -1,18 +1,18 @@
 type: object
 properties:
-  Name:
+  name:
     type: string
     example: Default
-  Ports:
+  ports:
     type: array
     items:
       $ref: ./HostServerPort.yaml
-  HA:
+  ha:
```

Comment:
HA --> High Availability? Does the API standard allow for exceptions when abbreviations are used? Or, maybe we should not be using abbreviations for property names -- :). Not asking for a change. 'ha' compared to 'HA" just seems odd.

Reply:
We do use other abbreviations though -- ip, dns. vlan, ...

Reply:
For camelCase, usually acronyms should be capitalized only if they are not at the beginning of the word, but in reality API developers have different interpretation 😄

Reply:
discussed with Dennis, and will be following `acronyms should be capitalized only if they are not at the beginning`